src/analysis/normed/group/seminorm.lean

```diff
@@ -344,6 +436,76 @@ ext $ λ x, real.smul_max _ _
 
 end add_group_seminorm
 
+namespace nonarch_add_group_seminorm
+section add_group
+variables [add_group E] [add_group F] [add_group G] {p q : nonarch_add_group_seminorm E}
+
+instance nonarch_add_group_seminorm_class :
+  nonarch_add_group_seminorm_class (nonarch_add_group_seminorm E) E :=
+{ coe := λ f, f.to_fun,
+  coe_injective' := λ f g h, by cases f; cases g; congr',
+  map_add_le_max := λ f, f.add_le_max',
+  map_zero := λ f, f.map_zero',
+  map_neg_eq_map' := λ f, f.neg', }
+
+/-- Helper instance for when there's too many metavariables to apply `fun_like.has_coe_to_fun`. -/
+instance : has_coe_to_fun (nonarch_add_group_seminorm E) (λ _, E → ℝ) := ⟨to_fun⟩
```

Comment:
```suggestion
instance : has_coe_to_fun (nonarch_add_group_seminorm E) (λ _, E → ℝ) :=
fun_like.has_coe_to_fun
```

Reply:
If I make this change, I get the error 
```
missing 'noncomputable' modifier, definition 'has_coe_to_fun' depends on 'real.linear_order'
```
(and similar errors on instances that depend on this).

Reply:
Ah I know, that's because the `fun_like` goes through some `linear_ordered_hom_class`, which depends on `real.linear_order`. A bit annoying so I guess you can keep it like that.